FirebaseAuth/Sources/Swift/AuthProvider/PhoneAuthCredential.swift

```diff
@@ -20,57 +20,45 @@ import Foundation
  */
 @available(iOS 13, tvOS 13, macOS 10.15, macCatalyst 13, watchOS 7, *)
 @objc(FIRPhoneAuthCredential) public class PhoneAuthCredential: AuthCredential, NSSecureCoding {
-  // TODO: delete objc's and public's below
-  @objc public let temporaryProof: String?
-  @objc public let phoneNumber: String?
-  @objc public let verificationID: String?
-  @objc public let verificationCode: String?
+  enum CredentialKind {
+    case phoneNumber(String, String) // phoneNumber, temporaryProof
+    case verification(String, String) // id, code
```

Comment:
If you didn't already try this and prefer it, you should be able to do:
```swift
    case phoneNumber(phoneNumber: String, temporaryProof: String)
    case verification(id: String, code: String)
```

Reply:
Thanks!

Going with 
```swift
    case phoneNumber(_ phoneNumber: String, _ temporaryProof: String)
    case verification(_ id: String, _ code: String)
```
 in the subsequent PR

Reply:
In #11445